src/main/java/sirius/biz/storage/layer3/VirtualFile.java

```diff
@@ -81,6 +82,7 @@ public abstract class VirtualFile extends Composable implements Comparable<Virtu
     private static final String HANDLER_CONSUME_FILE_HANDLER = "consumeFileHandler";
     private static final String MESSAGE_KEY_LOAD_FROM_URL_FAILED = "$VirtualFile.loadFromUrlFailed";
     private static final String MESSAGE_KEY_LOAD_FROM_URL_DISABLED = "$VirtualFile.loadFromUrlDisabled";
+    private static final int NUMBER_OF_RETRIES_FOR_SERVICE_UNAVAILABLE = 3;
```

Comment:
This will make BMEcat Imports in OX take even longer. I wonder if the number of retries and wait period between them should be customizable...

Reply:
Do you get a lot of 503s?

Reply:
Don't have a number. I guess practice will tell...